data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -10,6 +10,59 @@ local root_filesystem(filesystem) = {
   ],
 };
 
+local resize_vda() = {
+  drives: [
+    {
+      "search": "/dev/vda",
+      "partitions": [
+        {
+          "search": "/dev/vda2",
+          "filesystem": { "path": "/" },
+          "size": "5 GiB",
```

Comment:
we are not specifying the type of filesystem, 5GiB is the minimum for non btrfs. As we are picking the default system as starting point, I think we need minimum 12.5 GiB.